pkgs/by-name/xv/xvidcore/package.nix

```diff
@@ -31,7 +31,12 @@ stdenv.mkDerivation rec {
     [ ]
     # Undocumented darwin hack (assembly is probably disabled due to an
     # issue with nasm, however yasm is now used)
-    ++ lib.optional stdenv.hostPlatform.isDarwin "--enable-macosx_module --disable-assembly";
+    ++ lib.optionals stdenv.hostPlatform.isDarwin [
+      "--enable-macosx_module"
+    ]
+    ++ lib.optional (stdenv.hostPlatform.isDarwin || stdenv.hostPlatform.isFreeBSD) [
```

Comment:
```suggestion
    ++ lib.optionals (stdenv.hostPlatform.isDarwin || stdenv.hostPlatform.isFreeBSD) [
```

Reply:
Fixed. That's the problem with mass-producing pull requests like this...